snapcraft/plugins/colcon.py

```diff
@@ -643,6 +652,11 @@ def _build_colcon_packages(self):
         # Finally, add any cmake-args requested from the plugin options
         colconcmd.extend(self.options.colcon_cmake_args)
 
+        if self.options.colcon_packages_ignore:
```

Comment:
This is a bit of a nit, but can you move this above the `--cmake-args`? I'm not sure how they're written those consuming types of flags and would rather avoid this option being eaten by that one. It probably makes the most sense to have it directly following `--packages-select`.

Reply:
sure, I will move it